tf/BUILD.tpl

```diff
@@ -0,0 +1,18 @@
+package(default_visibility = ["//visibility:public"])
```

Comment:
Is the tf/ dir intended strictly for TensorFlow deps? Perhaps a more explicit name -- or a more general one if we anticipate needing other deps here eventually -- would help people navigate?

Reply:
The directory name was just copied from [custom-op](https://github.com/tensorflow/custom-op). I agree though it's not really describing what it does. Perhaps `tf_dependency`?

Reply:
Ah, if it's copied over, probably best to leave as-is for now for consistency. @yifeif - maybe a more explicit name for that would help?

Reply:
Although now I see you have updated it already; either way is fine by me.